prog/dftb+/prg_dftb/initprogram.F90

```diff
@@ -654,6 +654,17 @@ module initprogram
   !> Whether atomic coordindates have changed since last geometry iteration
   logical :: tCoordsChanged
 
+  !c3091013
+  !Are we using plumed?
+  logical :: tPlumed
+  !conversion factors for plumed
+  !conversion factor from kj/mol to hartree
+  real(dp), parameter :: KJM_TO_H = 2.62635e3_dp
+  !conversion factor from nm to bohr
+  real(dp), parameter :: NM_TO_B = 5.29177e-2_dp
+  !converstion factor from ps to au
+  real(dp), parameter :: PS_TO_AU = 2.41888e-5_dp
+
```

Comment:
Those all belong into `constants.F90`. (Also keep case convention there, although they deviate from the official one..)

Reply:
Another concern is to make the values consistent with those already in use in constants.F90 (there are also only a few decimal places in your inserted values). Is Plumed internally making any conversions, so again there may be concern about consistent units there.